packages/account/src/Components/poi/status/unsupported/card-details/selfie-upload.tsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import { Formik, Form } from 'formik';
+import { Formik, Form, FormikProps, FormikValues } from 'formik';
 import { localize } from '@deriv/translations';
 import { isMobile } from '@deriv/shared';
 import { Button, Icon, Text } from '@deriv/components';
 import Uploader from './uploader.jsx';
 import { setInitialValues, validateFields } from './utils';
 import { ROOT_CLASS, SELFIE_DOCUMENT } from '../constants';
 
-const SelfieUpload = ({ initial_values, goBack, onConfirm, onFileDrop }) => {
+type TSelfieUpload = {
+    initial_values: object;
```

Comment:
Can this be pf type `FormikValues`? Since this object is assigned as initial values to Formik

Reply:
sure will change it 👌